### DISCUSSION TITLE
Issue with Using Writable Views and Dynamic Form Modal for Data Input

### DISCUSSION THREAD

**Discussion by davidgarra** [mia-platform/community#368] [Q&A] [answered]
### Which section?

Backoffice

### Which is the version?

back-kit@1.4.7, crud-service@6.9.4

### Description

I'm struggling to figure out how to use the Writable Views with a Dynamic Form Modal for data input, as I can't seem to find the necessary instructions in the documentation

This is what I've done:
1. I created a new View `prescriptions_view` with the Writable flag set to true
2. I created the corresponding endpoint `/prescriptions-view` (here I see only the get methods...)
3. I added the Dynamic Form Modal with the basePath `/prescriptions-view`
4. I added the `bk-crud-client` with the basePath `/prescriptions-view` and the same dataSchema of the Dynamic Form Modal

Unfortunately, the POST during the add doesn't works, returning the http status code 405.

Is there any step that I missed to do?

**Comment by clabene** [comment]
Hi @davidgarra,

thanks for reaching out.

Please manually add missing routes for the endpoint `/prescriptions-view`:
  - method: DELETE, path: "/"
  - method: PATCH, path: "/"
  - method: POST, path: "/"
  - method: GET, path: "/lookup/<lookup field>/" (for each lookup field)

We are working on automatic creation of such routes.
In the meantime, however, we are going to integrate our documentation with the provided instructions to facilitate the process of creating Writable Views.

**Reply by davidgarra** [reply]
Thanks for the response.
I tried creating the endpoint manually as you can see:
<img width="893" alt="image" src="https://github.com/mia-platform/community/assets/53945399/4f4f11bf-d3fa-4322-8bc0-8c721f82df1e">
Then after the deploy I receive the error `404` during the POST to `/bo-prescriptions-drugs-view/`

```json
{
    "message": "Route POST:/bo-prescriptions-drugs-view/ not found",
    "error": "Not Found",
    "statusCode": 404
}
```

**Reply by davidgarra** [reply, marked as answer]
I've found where the problem is, the internal-endpoint e the endpoint should be the same, so for everyone who has the same problem: pay attention to use the same name during the view creation
